### PR TITLE
add brew formula for pangolin

### PR DESCRIPTION
**Pull request by JohnN193**

### CODE REVIEW

File: Formula/pangolin.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+class pangolin < Formula
```
Comment (Otterverse) [inline Formula/pangolin.rb:1]: This fails to run:

```
Error: pangolin: /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/viamrobotics/homebrew-brews/Formula/pangolin.rb:1: class/module name must be CONSTANT
class pangolin < Formula
```
You have to a capitalized class name.

File: Formula/pangolin.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  depends_on "lz4"
+  depends_on "zstd"
+  depends_on "libxkbcommon"
+  
+  def install
+    system "sed", "-i.bak", "/FF_API_XVMC/,+3d", "components/pango_video/include/pangolin/video/drivers/ffmpeg_common.h"
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+
```
Comment (Otterverse) [inline Formula/pangolin.rb:26]: [nit] No empty line

**Review by Otterverse — CHANGES_REQUESTED** [review]
Two small things to fix...

**Comment by JohnN193** [comment]
when auditting was getting the error 
```
Failed to load cask: pangolin.rb
Cask 'pangolin' is unreadable: wrong constant name #<Class:0x000000012972eb68>
```
otherwise made changes based off the output from the audit

**Comment by Otterverse** [comment]
> when auditting was getting the error
> 
> ```
> Failed to load cask: pangolin.rb
> Cask 'pangolin' is unreadable: wrong constant name #<Class:0x000000012972eb68>
> ```
> 
> otherwise made changes based off the output from the audit

You're not maintaining a cask, (prebuilt package) so I think that's okay.

**Review by Otterverse — APPROVED** [review]
LGTM!

**Pull request merged**